This will prevent adding more usages of unsafe methods

tgolen opened the pull request:
cc @fnwbr

marcaaron commented:
Any test steps for this? I made eslint-config-expensify dependency in web-e point to the last sha on this branch, but could not get this error to show.

tgolen commented:
heh, I didn't test it and just assumed it would work. Let me take a look

On Fri, Jan 31, 2020 at 2:22 PM Marc Glasser <notifications@github.com>
wrote:

> Any test steps for this? I made eslint-config-expensify dependency in
> web-e point to the last sha on this branch, but could not get this error to
> show.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Expensify/eslint-config-expensify/pull/14?email_source=notifications&email_token=AAJMABZFUEKQKSOZGAA4D4LRASQB3A5CNFSM4KOOEG52YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEKQGKSQ#issuecomment-580937034>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJMAB4RGVMNIJZTFLJ5URTRASQB3ANCNFSM4KOOEG5Q>
> .
>

tgolen commented:
OK, this PR is ready to go.

Thanks to Florian's excellent findings about React version support, I've gone and updated all versions of dependencies, and that's got it working.

![image](https://user-images.githubusercontent.com/1228807/73593566-6c4fdb00-44ba-11ea-9b43-8669d14baa36.png)

1. I moved one rule from web-e over here, which shouldn't have ever been placed there
1. I have modified a rule for `arrow-parens` which was a change I saw from the newest version of airbnb and keeps it more in line with our current code style which I prefer.
